app/controllers/review_mapping_controller.rb

```diff
@@ -51,129 +47,196 @@ def select_metareviewer
     @mapping = ResponseMap.find(params[:id])
   end
 
+
+  #EDITED: If the user is invalid(not present in database), then return with error message.
+  #If reviewer is a participant of the assignment, then only move forward with reviewer assignment.
```

Comment:
Please reword this comment.  Should it be something like, "A user may only be assigned to review someone if that reviewer is a participant in the assignment."?  But it still is not very clear which lines of code enforce this provision.